config/crds/bootstrap.go

```diff
@@ -177,18 +177,18 @@ func CreateSingle(ctx context.Context, client apiextensionsv1client.CustomResour
 	})
 }
 
+func retryRetryableErrors(f func() error) error {
+	return retry.OnError(retry.DefaultBackoff, func(err error) bool {
+		return utilnet.IsConnectionRefused(err) || apierrors.IsTooManyRequests(err) || apierrors.IsConflict(err)
+	}, f)
+}
+
 // Unmarshal YAML-decodes the give embedded file name into the target.
-func Unmarshal(fileName string, crd *apiextensionsv1.CustomResourceDefinition) error {
+func Unmarshal(fileName string, o interface{}) error {
```

Comment:
why not `crd *apiextensionsv1.CustomResourceDefinition` ?
it looks like all files in this directory are CRDs

Reply:
oh, I thought that came in through rebase. I can remove the change.

Reply:
done